src/main/cpp/t08_queen.cpp

```diff
@@ -19,10 +19,59 @@
 
 #include <iostream>
 #include <vector>
-
+#include <iomanip>
+#include <cmath>
 using namespace std;
+int count = 0;
 
-int t08_queen(){
+void ferz(int x, int y, vector < vector <int> > a, int n) {  
+	int j;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'j' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4504343)